### PR TITLE
Subtask wave2 795 global banner brick new fields char count required fields

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: themes/seven/style.css [hunk 1]
```diff
@@ -1006,4 +1009,10 @@ div.add-or-remove-shortcuts {
   border-radius: 8px;
 }
 
+#field-uc-banner-bg-color-add-more-wrapper label:after {
```
Comment (dpfitzsi) [inline themes/seven/style.css]: This CSS should be included as part of the hook_css_alter() that will be added to ucgy_core_layout.module to reference your own copy of the seven.css file.

Comment (mhughes2012) [inline themes/seven/style.css]: fixed in PR 406

**Comment by dpfitzsi** [comment]
Hi Mike, I have merged in 797 and now this is in conflict. Can you resolve?

**Comment by mhughes2012** [comment]
Hi @dpfitzsi, resolved.

**Pull request merged**